src/printer.js

```diff
@@ -451,20 +473,25 @@ const embed = (path, print, textToDoc, opts) => {
     return textToDoc(node.content, { ...opts, parser: 'html' });
   }
 
-  return null;
```

Comment:
TBH I was a bit confused on this. I found that returning `null` in certain places broke the printer, so I just assumed that there was a problem with it. But it may just be a problem with our own printer! I can revert this.